vita3k/gui/src/controls_dialog.cpp

```diff
@@ -15,45 +15,276 @@
 // with this program; if not, write to the Free Software Foundation, Inc.,
 // 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-#include <gui/functions.h>
-
 #include "private.h"
 
+#include <config/functions.h>
+#include <gui/functions.h>
 #include <host/state.h>
+#include <interface.h>
+
+
 
```

Comment:
You see I am the pickiest man on the planet and 1 space is okay, 2 spaces is pushing it but 3 spaces!?!?!? format.sh should've handled this too lol